gcp/table_gcp_sql_database_instance_metric_connections.go

```diff
@@ -0,0 +1,47 @@
+package gcp
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+	sqladmin "google.golang.org/api/sqladmin/v1beta4"
+)
+
+//// TABLE DEFINITION
+
+func tableGcpSQLDatabaseInstanceConnectionsMetric(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "gcp_sql_database_instance_metric_connections",
+		Description: "GCP SQL Database Instance Metrics Connections",
```

Comment:
same here, should it be `gcp_sql_database_instance_connections`

Reply:
@rajmohanty17 We have followed same naming pattern defined in AWS